Libraries/src/Amazon.Lambda.MQEvents/ActiveMQEvent.cs

```diff
@@ -78,6 +78,11 @@ public class ActiveMQMessage
             /// Broker Out Time
             /// </summary>
             public long? BrokerOutTime { get; set; }
+
+            /// <summary>
+            /// Custom Properties
+            /// </summary>
+            public Dictionary<string, string> Properties { get; set; }
```

Comment:
Can you make this `IDictionary` instead of `Dictionary`?

Reply:
Done, thank you!

Reply:
Let me know if you need anything else